libs/config/common/src/lib/endpointLoader.ts

```diff
@@ -83,8 +83,6 @@ export class EndpointLoader {
 
         this.endpointsTemplate.platformApp = dynamicEndpoints?.platformApp || 'https://biosimulations.dev';
 
-        this.endpointsTemplate.dispatchApp = 'https://run.biosimulations.org';
-
```

Comment:
why remove only the endpointsTemplate.dispatchApp URLs and not the others?  shouldn't this one be a 'dynamiccEndpoint' also?

Reply:
Great question. As you can see, I have set the endpoint for dispatch explicitly above (which sets to `.dev`). I then choose to set it with a different value explicitly here. Since we were iterating over the same schema for `endpointsTemplate`, setting a repeated value like this with a method saves code duplication, etc. Only this field is originally targeted as it was pertinent to our concerns, but it is a chore of mine to set all of these fields dynamically.